src/components/AudioPlayer/components/PlaybackSpeedPopup.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react'
+import { Popover } from 'antd'
+import { Button } from '../../Button/index'
+
+const speeds = [0.8, 1, 1.2, 1.5, 1.7, 2, 2.5]
+export default function PlaybackSpeed({ waveSurfer }) {
+  const [speed, setSpeed] = useState(waveSurfer?.getPlaybackRate())
+
+  const handleSpeedChange = (speed) => {
```

Comment:
handleSpeedChange can be reused from src/components/AudioPlayer/components/PlaybackSpeed.js

Reply:
moved it to utils and reusing from there